simp_le.py

```diff
@@ -689,13 +693,11 @@ def load(self):
             cert, chain = None, None
         else:
             cert, chain = data.chain[0], data.chain[1:]
-        return self.Data(account_key=data.account_key, key=data.key,
-                         cert=cert, chain=chain)
+        return self.Data(cert=cert, chain=chain)
```

Comment:
Reusing `data.account_key` or `data.key` was intentional - it plays well with the inheritance